dev-client/src/types/projects.types.ts

```diff
@@ -1,5 +1,11 @@
 import {UserPreview} from './users.types';
 import {SitePreview} from './sites.types';
+
+export interface ProjectName {
+  name: string;
+  id: string;
+}
```

Comment:
question: what is this type for?

Reply:
This was just something I did to make typescript happy at one point. It's only used for dummy data, so at some point it will be deleted. I'll make a tech-debt issue to delete all of our dummy data at some point.